web/packages/teleport/src/ironrdp/src/lib.rs

```diff
@@ -285,7 +284,7 @@ fn extract_smallest_rectangle(
 ) -> (InclusiveRectangle, Vec<u8>) {
     let pixel_size = usize::from(image.pixel_format().bytes_per_pixel());
 
-    let image_width = usize::try_from(image.width()).unwrap();
+    let image_width = image.width() as usize;
```

Comment:
`as` conversions are generally evil (because they work too much).

```suggestion
    let image_width = usize::from(image.width());
```

Reply:
Thanks! Clippy suggested `From::from` and I didn't know what it was asking me to do. 🤷